lib/rummager_finder_publisher.rb

```diff
@@ -31,6 +31,9 @@ def pre_production?(schema)
 
   def export_finder(schema)
     presenter = FinderRummagerPresenter.new(schema[:file], schema[:timestamp])
-    RummagerWorker.perform_async(presenter.type, presenter.id, presenter.to_json)
+
+    logger.info("Publishing '#{schema[:file]['name']}' finder")
+
+    Services.rummager.add_document(presenter.type, presenter.id, presenter.to_json)
```

Comment:
I wouldn't say that because you trigger this manually it shouldn't be an asynchronous operation. If there was lots of records then it would be beneficial to have Sidekiq performing the job.